app.js

```diff
@@ -3,12 +3,17 @@ const hashString = 'Hello from IPFS Gateway Checker'
 
 const $results = document.querySelector('#results')
 
+function returnHtmlLink (gateway) {
+	let gatewayTitle = gateway.split(hashToTest)[0]
+	return '<a title="' + gatewayTitle + '" href="' + gateway '" >' + gateway + '</a>';
```

Comment:
It's missing a `+` here I think.

Would probably be easier to write it like this:

return `<a title="${gatewayTitle}" href="${gateway}">${gateway}</a>`

Reply:
Updated previous comment as I made a mistake with the formatting.

Reply:
indeed, thank you